README.md

```diff
@@ -11,27 +11,54 @@ Here is some material to read up on DLCs and ItchySats' protocol implementation:
 - [Blogpost](https://comit.network/blog/2022/01/11/cfd-protocol-explained) by @luckysori of the COMIT team, describing the protocol in detail
 - [DLC overview](https://bitcoinops.org/en/topics/discreet-log-contracts/) on Bitcoin Optech for information beyond ItchySats
 
+## Installation
+
+### Bitcoin/Lightning nodes
+
+ItchySats is available on many DIY Bitcoin/Lightning node distributions:
+
+- [RaspiBlitz](https://github.com/rootzoll/raspiblitz#feature-overview)
+- [Umbrel](https://getumbrel.com/)
+- [Citadel](https://runcitadel.space/)
+
+### Desktop App
+
+We package ItchySats as an Electron app for Windows and MacOS.
+Latest release can be found [here](https://github.com/itchysats/itchysats/releases).
+
+### Homebrew (macOS)
+
+We maintain a [homebrew](https://brew.sh/) tap with the latest version of ItchySats.
+
+```sh
+brew tap itchysats/itchysats
+brew install itchysats
+```
+
+After initial install, homebrew will take care of keeping ItchySats up-to-date.
+
+### Mobile
+
+ItchySats is not available as a full mobile app yet.
+In the meantime, if you'd like to use ItchySats on-the-go, the best way is to install ItchySats on your bitcoin node and use
+[Tailscale](https://tailscale.com/) or connect to your node via Tor.
```

Comment:
I know we have it somewhere else but personally I don't feel confident with recommending a service I have not evaluated. If you would trust it, I'm happy to leave it here though.

Reply:
I totally recommend evaluating it, the experience is way better than over Tor.
I use it myself, and also heard this being recommended as the go-to way to connect to your umbrel node by other members of the community, including Specter wallet dev at his talk at the latest conference.